code/modules/hydroponics/grown/citrus.dm

```diff
@@ -33,6 +33,18 @@
 	filling_color = "#00FF00"
 	juice_results = list(/datum/reagent/consumable/limejuice = 0)
 
+/obj/item/reagent_containers/food/snacks/grown/citrus/lime/attackby(obj/item/item, mob/user, params)
+	if(!istype(item ,/obj/item/kitchen/knife) && INTENT_HARM)
```

Comment:
what are you doing here? INTENT_HARM is a string. you are checking if the item isnt a knife and if the string "harm" is truthy (which is always)

Reply:
mothbrain moment, fixed this